lab-raziyeh/data.json

```diff
@@ -0,0 +1,3 @@
+{
+  "text":"raziyeh"
+}
```

Comment:
@Raziyehbazargan Got the new line issue again - this could be in relation to your editor.  Lets discuss tomorrow morning before class/lab and get this fixed, in the event it's your editor.